src/frontendFixes/Fixes/underlineFix.js

```diff
@@ -0,0 +1,50 @@
+const ForceUnderlineFixData = window.edac_frontend_fixes?.underline || {
+	enabled: false,
+};
+
+const ForceUnderlineFix = () => {
+	if ( ! ForceUnderlineFixData.enabled ) {
+		return;
+	}
+
+	// Apply underline to any link that is not within a `nav` element.
+	// Using JavaScript ensures this style takes precedence over conflicting CSS rules.
+	const targets = document.querySelectorAll( ForceUnderlineFixData.target );
+	const originalOutline = targets[ 0 ].style.outlineWidth;
+	const originalOffset = targets[ 0 ].style.outlineOffset;
+	const textColor = targets[ 0 ].style.color;
+	const originalColor = targets[ 0 ].style.outlineColor;
```

Comment:
Would storing these originals on each element be better in the `focusin` event? Maybe put them in `data-*` attributes when swapped so that each targets own original styles could be restored?

Reply:
Yeah, this would be better. I've pushed an update.